nbs/models.tsmixerx.ipynb

```diff
@@ -55,8 +55,6 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #236.                # x = x.reshape(batch_size, self.h, -1)               #   [B, h, n_outputs, N] -&gt; [B, h, n_outputs * N]
</pre><p>is this commented line something to review later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Good catch, that's redundant, I'll remove that